seq2science/envs/kb_seurat_pp.yaml

```diff
@@ -0,0 +1,22 @@
+name: kb_seurat_pp
+channels:
+  - r
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies: 
+  - r-base>=3
+  - r-rmarkdown
+  - r-knitr>=1.20
+  - bioconductor-biocinstaller>=1.30
+  - r-devtools>=2
+  - r-matrix
+  - r-irlba
+  - r-dplyr
+  - r-ggplot2>=3.0
+  - r-mvoutlier
+  - bioconductor-singlecellexperiment
+  - bioconductor-scran
+  - bioconductor-scater>=1.10
+  - r-seurat>=3
+  - r-plot3d

```

Comment:
Sorry I missed this originally. But it's best to pin each version to a specific version. We don't want workflows to stop working because of some incompatible changes upstream